### PR TITLE
[onert] Add Shape inference for FullyConnected

### PR DESCRIPTION
**Pull request by YiHyunJin**
This PR adds shape inference for FullyConnected op.

Signed-off-by: YiHyunjin <hj0412.yi@samsung.com>

### CODE REVIEW

**Comment by YiHyunJin** [comment]
If merge https://github.com/Samsung/ONE/pull/1821, It will work.

**Comment by glistening** [comment]
@nnfw-bot test cross-aarch64-runtime-test-release

File: runtime/onert/core/src/util/shapeinf/FullyConnected.cc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+  // if input is dynamic, output also becomes dynamic
+  if (input.info().isDynamic())
+  {
+    output.info().setDynamic();
+    return;
+  }
+
+  const auto ker_idx{op.getInputs().at(ir::operation::FullyConnected::Input::WEIGHT)};
+  const auto &ker = _operands.at(ker_idx);
```
Comment (hyunsik-yoon) [inline runtime/onert/core/src/util/shapeinf/FullyConnected.cc:55]: If ker is always from constant, I think this code works well..
hmm.. can ker be from output of other op? 
if we want ker to be constant, maybe some `throw` (when ker is not constant) is needed.
if we want ker to be output of other op, I think that `ker.info().is_dynamic()` also need to be checked.

**Comment by hyunsik-yoon** [comment]
Could you please add one more contributor in WG2 as a reviewer? Thanks!

File: runtime/onert/core/src/util/shapeinf/FullyConnected.cc (hunk 1 above)
Comment (glistening) [reply runtime/onert/core/src/util/shapeinf/FullyConnected.cc:55]: For shape inference, kernel doen't need to be constant. If the shape of kernel is determined, it is okay. We may check whether kernel `hasUnspecifiedDims`. I am not sure `isDynamic` is necessary.

**Pull request merged**